styles/variables-silicon.less

```diff
@@ -8,6 +8,18 @@
 // to this file with the real value for the "silicon" skin.
 //
 
+// Common
+// --------------------------------
+@agate-base-font-family: "LGSmartUI";
```

Comment:
`LGSmartUI` is absolutely needed for our GUI. But the `LGSmartUI` font is allowed to only our company products. Agate is used in webOS OSE (webOS open source edition ), so it could be used in other products. Therefore we need to find the way to use the `LGSmartUI` out of the Agate.